src/main/sensors/gyroanalyse.c

```diff
@@ -152,8 +163,9 @@ void gyroDataAnalyse(gyroAnalyseState_t *state, biquadFilter_t *notchFilterDyn)
         // calculate mean value of accumulated samples
         for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
             float sample = state->oversampledGyroAccumulator[axis] * state->maxSampleCountRcp;
-            sample = biquadFilterApply(&state->gyroBandpassFilter[axis], sample);
-
+            if (gyroConfig()->dyn_notch_quality > 4){
```

Comment:
Space between `)` and `{`. I think we discussed this should be handled differently, with 0 being the special value.